src/Thinktecture.Relay.Abstractions/StreamExtensions.cs

```diff
@@ -9,6 +9,22 @@ namespace Thinktecture.Relay
 	/// </summary>
 	public static class StreamExtensions
 	{
+		/// <summary>
+		/// Sets the position of the stream to zero, if the stream supports seeking.
+		/// </summary>
+		/// <param name="stream">The <see cref="Stream"/> that will be rewind.</param>
+		/// <returns>True, if rewinding was successful; otherwise, false</returns>
```

Comment:
```suggestion
		/// <param name="stream">The <see cref="Stream"/> that will be rewinded.</param>
		/// <returns>true, if rewinding was possible; otherwise, false</returns>
```